Header header
float64 Latitude
float64 Longitude
float64 Altitude
float64 Utm_easting
float64 Utm_northing
string Zone
string ZoneLetter